js/huobi.js

```diff
@@ -3199,9 +3196,9 @@ module.exports = class huobi extends Exchange {
                 account['used'] = this.safeString (balance, 'margin_frozen');
                 result[code] = account;
             }
+            result = this.safeBalance (result);
         }
-        const isolatedMargin = isolated && (spot || margin);
-        return isolatedMargin ? result : this.safeBalance (result);
```

Comment:
these two lines were vague, instead, i've moved the `.safeBalance` to places where it was needed.